src/components/PurchaseOrder/PO.js

```diff
@@ -155,21 +158,44 @@ class PO extends Component {
     </Button>
   );
 
+  mountModal = () => {
+    this.setState({ isRenderModal: true });
+  }
+
+  unmountModal = () => {
+    this.setState({ isRenderModal: false });
+  }
+
   render() {
     const { location, history, match, mutator, resources, parentResources } = this.props;
     const order = get(resources, ['order', 'records', 0]);
     const orderId = get(order, 'id');
     const orderNumber = get(order, 'po_number', '');
     const poLines = get(order, 'po_lines', []);
+    const workflowStatus = get(order, 'workflow_status', '');
+    const isVisible = orderId && workflowStatus === WORKFLOW_STATUS.open;
```

Comment:
`isVisible` is a very common name, I think it could be more specific for better readability

Reply:
done